src/openssl/bio.cr

```diff
@@ -34,31 +47,56 @@ struct OpenSSL::BIO
       LibCrypto::Long.new(val)
     end
 
-    crystal_bio.create = LibCrypto::BioMethodCreate.new do |bio|
-      bio.value.shutdown = 1
-      bio.value.init = 1
-      bio.value.num = -1
+    create = LibCrypto::BioMethodCreate.new do |bio|
+      {% if LibCrypto::OPENSSL_110 %}
+        LibCrypto.BIO_set_shutdown(bio, 1)
+        LibCrypto.BIO_set_init(bio, 1)
+        # bio.value.num = -1
```

Comment:
I have no idea what `BIO#num` means and what we should do here. Anybody knows?

Reply:
It seems to come originally from https://github.com/crystal-lang/crystal/commit/4995fb010316ac0cb3085ab18f8738f7b7415140#diff-65c7a3575f24669104f83007f62e90c6R109 if the documentation of 1.1.x says nothing about it it's fine as it is I guess.